32_flask/templates/book.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Book</title>
+    <link rel="stylesheet" href="{{ url_for('static', filename='books.css') }}">
+</head>
+<body>
```

Comment:
use base.html template with all similar structure and extend it in all other templates

Reply:
ok, i change this later